src/main/java/uk/gov/service/notify/NotificationClient.java

```diff
@@ -310,9 +311,21 @@ public static JSONObject prepareUpload(final byte[] documentContents) throws Not
 
         JSONObject jsonFileObject = new JSONObject();
         jsonFileObject.put("file", fileContent);
+        jsonFileObject.put("is_csv", isCsv);
         return jsonFileObject;
     }
 
+    /**
```

Comment:
Chosen a method overload here but no idea if that is the best way to do a default argument (see https://stackoverflow.com/questions/997482/does-java-support-default-parameter-values).

Also I'm not sure if this is how to do the doc block by keeping the same description as the un overloaded method above

Reply:
It's right to overload the method, that way it doesn't affect anyone using origin method

Reply:
It's right to overload the method. It won't cause a problem for people using the original method. 
The comments are fine.